src/librustc/lint/builtin.rs

```diff
@@ -318,6 +318,12 @@ declare_lint! {
     "warn about missing code example in an item's documentation"
 }
 
+declare_lint! {
+    pub PRIVATE_DOC_TEST,
```

Comment:
The lint name doesn't follow [conventions](https://github.com/rust-lang/rfcs/blob/master/text/0344-conventions-galore.md#lints).

Reply:
Didn't know there was one, thanks!